modules/ROOT/pages/sharepoint/sharepoint-connector.adoc

```diff
@@ -1,9 +1,9 @@
-= Microsoft SharePoint Connector
+= Microsoft SharePoint Connector - Mule 4
```

Comment:
Edit to green text only.
If you need me to perform a more extensive edit, please resubmit when able. Create a JIRA to remember.